### PR TITLE
Static table download csv

### PR DESCRIPTION
**Pull request by jstirling91**
Add a button to download the current table as a csv file

---
This change is [Reviewable](https://reviewable.io/reviews/aces/loris/1472)

### CODE REVIEW

**Comment by driusan** [comment]
Needs to be rebased, but otherwise LGTM.

File: jsx/StaticDataTable.js [hunk 1]
```diff
@@ -49,6 +49,31 @@ StaticDataTable = React.createClass({
            'PageNumber' : 1
        });
     },
+    downloadCSV: function() {
+        var headers = this.props.Fields,
+            csvworker = new Worker('js/workers/savecsv.js');
```
Comment (driusan) [inline jsx/StaticDataTable.js]: this should include loris.BaseURL. Otherwise it won't work from within a module..

**Comment by xlecours** [comment]
It is working with Chrome but not with Firefox. Is it only on my workstation?

**Comment by johnsaigle** [comment]
The code looks good, but it says the branch has conflicts

**Comment by xlecours** [comment]
This work with both Firefox 45.0 and Chrome 49.0.2623.87

**Pull request merged**